sage-ios/SAGE/SAGE/AddSchoolRadiusView.swift

```diff
@@ -0,0 +1,111 @@
+//
+//  AddSchoolRadiusView.swift
+//  SAGE
+//
+//  Created by Erica Yin on 2/21/16.
+//  Copyright © 2016 Cal Blueprint. All rights reserved.
+//
+
+import Foundation
+
+class AddSchoolRadiusView: UIView {
+    
+    let leftMargin = CGFloat(35)
+    let sliderHeight = CGFloat(45)
+    let maximumRadius = Float(700)
+    var mapView: GMSMapView = GMSMapView()
+    var circle = GMSCircle()
+    var radiusCenter = CLLocationCoordinate2D()
+    var radius = CLLocationDistance(200)
+    var sliderView = UIView()
+    var slider = UISlider()
+    var radiusLabel = UILabel()
+    
+    init(frame: CGRect, center: CLLocationCoordinate2D, radius: CLLocationDistance?) {
+        super.init(frame: frame)
+        self.backgroundColor = UIColor.whiteColor()
+        self.radiusCenter = center
+        if radius != nil {
+            self.radius = radius!
+        }
+        self.slider.addTarget(self, action: "sliderValueDidChange:", forControlEvents: .ValueChanged)
+        self.setupSubviews()
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+    }
+    
+    func radiusToString(radius: Float) -> String {
+        var index = 1
+        if (radius >= 10 && radius < 100) {
+            index = 2
+        } else if (radius >= 100) {
+            index = 3
+        }
+        let roundedValueString: String = (String(radius) as NSString).substringToIndex(index)
```

Comment:
cant you just truncate to int soyou dont have to deal with any substring index stuff?